src/style/themer.scss

```diff
@@ -0,0 +1,199 @@
+/*Additional variables*/
+
+/*Dark Theme*/  
+html[data-theme='dark'] {
+
```

Comment:
Do the variables here have to be in scss and not css? We're currently not using scss in any of the apps so having everything in css is preferable

Reply:
Yeah, I'll test it out